### PR TITLE
Prepare 1.0.0 release.

### PR DESCRIPTION
**Pull request by Natim**
r? @ametaireau

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CHANGELOG
+=========
+
+This document describes changes between each past release.
+
+1.0.0 (2015-04-13)
+------------------
+
+- Add handler for Gecko Headless websockets.
```
Comment (ametaireau) [inline CHANGELOG.rst]: I would add this is the initial release.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+CHANGELOG
+=========
+
+This document describes changes between each past release.
+
+1.0.0 (2015-04-13)
```
Comment (ametaireau) [inline CHANGELOG.rst]: Are you sure you want to do a 1.0? Maybe a first round of release could let it mature.

File: README.rst [hunk 3]
```diff
@@ -13,81 +10,11 @@ Worker and talk with it through a websocket.
...
----------------------
-
-::
-
-    make mock_client
-
 
-Install Redis
-=============
+Remote Worker Server is the service that handle the signaling for the
```
Comment (ametaireau) [inline README.rst]: is a service which handles signaling for

File: README.rst [hunk 4]
```diff
@@ -13,81 +10,11 @@ Worker and talk with it through a websocket.
...
-    brew services restart redis
-
-
-Run tests
-=========
-
-::
-
-    make tests
+* `Initial Protocol Spec <https://github.com/mozilla-services/remote-worker-server/wiki/WebRTC-API-Design-Proposal>`_
```
Comment (ametaireau) [inline README.rst]: I'm not sure it's very useful to link to the initial protocol spec here. Maybe from the docs instead?

File: docs/api/authentication.rst [hunk 5]
```diff
@@ -0,0 +1,61 @@
+##############
+Authentication
+##############
+
+.. _authentication:
+
+We propose a setup using :term:`Firefox Accounts`, that verifies
```
Comment (ametaireau) [inline docs/api/authentication.rst]: I believe we should try to avoid "we" and "you" in the docs.

File: docs/api/client.rst [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+::
+
+    ws://localhost:8765/
+
+
+Creating a new worker
+=====================
+
+In order to ask for a worker creation client need to send this first standza:
```
Comment (ametaireau) [inline docs/api/client.rst]: "creation, the client needs"

File: docs/api/authentication.rst (hunk 5 above)
Comment (Natim) [inline docs/api/authentication.rst]: Yes, we should probably fix it here too then: http://cliquet.readthedocs.org/en/latest/api/authentication.html#authentication

File: docs/api/client.rst (hunk 6 above)
Comment (ametaireau) [inline docs/api/client.rst]: I'm not sure we should use the term "standza" (which is stanza, anyway). https://en.wikipedia.org/wiki/Stanza_%28disambiguation%29

File: docs/api/client.rst [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+* **candidate**: The candidate content
+* **sdpMid**: The candidate mid
+* **sdpLineIndex**: The SDP Line Index
+
+
+Receiving errors
+================
+
+In case the gecko is not available or was not able to start the
+worker, you will receive an error.
```
Comment (ametaireau) [inline docs/api/client.rst]: an error will be returned.

File: docs/api/client.rst [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+        "workerId": "<worker-id>",
+        "errno": "<worker error number>",
+        "reason": "<error reason>"
+    }
+
+
+Receiving WebRTC Answer
+=======================
+
+If everything worked, you should receive the Gecko Answer.
```
Comment (ametaireau) [inline docs/api/client.rst]: a gecko answer is sent back by the server.

File: docs/api/client.rst [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+        "action": "worker-hello",
+        "workerId": "<Worker ID >",
+        "webrtcAnswer": "<Gecko WebRTC Answer>"
+    }
+
+
+Receiving Gecko ICE Candidates
+==============================
+
+In order to setup the WebRTC data channel, you may receive the Gecko
```
Comment (ametaireau) [inline docs/api/client.rst]: It seems to make sense to define what's an ICE candidate.

File: docs/api/client.rst (hunk 6 above)
Comment (Natim) [inline docs/api/client.rst]: I will use message then

File: docs/api/client.rst (hunk 9 above)
Comment (Natim) [inline docs/api/client.rst]: Added in the glossary.

**Pull request merged**